### DISCUSSION TITLE
Best Optimization mod zo far

### DISCUSSION THREAD

**Discussion by zeridev** [xCollateral/VulkanMod#208] [General]
this is by far the best mod to increase performance in minecraft that i have seen in all my years of playing.
on my pc with a 3070 and ryzen 5 5600x it increased my fps on 32 render distance and max settings from an unstable unplayable experience with 80 -120 fps with a lot of stutters to a steady 300 - 400 fps.
even tho the mod is in alpha and there could still be optimizations left to further increase the performance i hope that the mod creator will continue to develop this mod to the point that minecraft will consider implementing vulkan as a secondary rendering engine.

**Comment by ghost** [comment]


**Reply by andreasdc** [reply]
> Now that Mojang has been controlled by Microsoft, Minecraft will never officially support Vulkan

For what reason?

**Reply by Splarkszter** [reply]
i would rather say: Microsoft would rather kill the java version to keep using their obnoxious DirectX API on Bedrock since they own that too.

Microsoft IS a monopolistic entity and we shall not let them do what they want. They also work with the glowies. Invading our privacy and human rights.

**Comment by andreasdc** [comment]
Sodium + ImmediatelyFast + many mods from Fabulously Optimized modpack = great performance, for today better than VulkanMod.

**Reply by Splarkszter** [reply]
For me, it's way easier on the CPU so i get more FPS while standing still and *also* when traveling (loading chunks).

**Reply by andreasdc** [reply]
Maybe depends on what you do, but when you see a lot of text and entities, huge difference.

**Reply by Splarkszter** [reply]
We will have to see when those mods get ported to the Vulkan API. Because even standalone it's a huge improvement. Something like exordium is always needed in any scenario.

**Reply by andreasdc** [reply]
They won't be ported, only chance is to get compability with them or making similar improvements for VulkanMod I think.

**Reply by sqrvrt** [reply]
Surprisingly, this is what is defined as "porting"

**Reply by andreasdc** [reply]
No, porting would be doing the same for VulkanMod.
